README.md

```diff
@@ -1,15 +1,15 @@
 # Memory Analysis Cube
 
-The Memory Analysis Cube (MAC) is an ActivePivot project that aims to provide
+The Atoti Memory Analysis Cube (MAC) is an Atoti project that aims to provide
 the necessary tools to explore and analyze the data consumption from any other
-ActivePivot application, starting from ActivePivot 5.8 onward.
+Atoti Server application, starting from Atoti Server 5.8 onward.
```

Comment:
maybe just an Atoti application :thinking: 
```suggestion
Atoti application, starting from Atoti Server 5.8 onward.
```
I think that we build Atoti applications using the server/runtime and adding the UI. But you can also create apps using the Atoti Python client. Etc.
Thus, using Atoti here seems more correct